src/Entity/TranslationRequestLog.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_translation\Entity;
+
+use Drupal\Core\Entity\ContentEntityBase;
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\Core\StringTranslation\TranslatableMarkup;
+
+/**
+ * Defines the Translation request log entity.
+ *
+ * @ContentEntityType(
+ *   id = "oe_translation_request_log",
+ *   label = @Translation("Translation request log"),
+ *   base_table = "oe_translation_request_log",
+ *   entity_keys = {
+ *     "id" = "id",
+ *     "label" = "id",
+ *     "uuid" = "uuid"
+ *   }
+ * )
+ */
+class TranslationRequestLog extends ContentEntityBase implements TranslationRequestLogInterface {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getCreatedTime(): string {
+    return $this->get('created')->value;
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function setCreatedTime(int $timestamp): TranslationRequestLogInterface {
+    return $this->set('created', $timestamp);
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getMessage(): TranslatableMarkup {
+    $text = $this->message->value;
+    // @codingStandardsIgnoreStart
+    if ($this->variables->first() && $this->variables->first()->toArray()) {
+      return new TranslatableMarkup($text, $this->variables->first()->toArray());
```

Comment:
I know tmtgmt translates this but we wont so no need for translatable markup. You can use FormattableMarkup.

Also, just check if !$this->variables->isEmpty()...